src/Chapter1.hs

```diff
@@ -620,7 +625,12 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n =
+  let m = abs n
+      last1 = mod m 10
```

Comment:
Neat solition! 👌🏼 
You can also notice that this is the `lastDigit` function from the previous task 🐱